domain_menu_access.module

```diff
@@ -41,17 +46,14 @@ function domain_menu_access_form_menu_link_content_form_alter(&$form, FormStateI
       '#open' => TRUE,
       '#weight' => 25,
     );
-
+  
+    $domains = Domain::loadMultiple();
+    $options = array_map(function($domain) {
+      return $domain->getDomainId();
```

Comment:
are you sure that domainId() should be there, I bet it should different + totally useless now